PlaceRouteHierFlow/thirdparty/SymCMakeFiles/CMakeLists.Cgl

```diff
@@ -0,0 +1,274 @@
+get_ac_init_version(${Cgl_DIR}/configure.ac CGL)
```

Comment:
@srini229 We shouldn't put these files in our ALIGN-public repo because they have the wrong license. We certainly can't have them in the repo and then change something. 
We need to have these in a different repo. We could add a symbolic link during our build flow.

Reply:
@stevenmburns Can I create a repo under the https://github.com/ALIGN-analoglayout hierarchy parallel to ALIGN-public to host these CMakeFiles?